dojo/unittests/test_spotbugs_parser.py

```diff
@@ -0,0 +1,16 @@
+from django.test import TestCase
+from dojo.tools.spotbugs.parser import SpotbugsXMLParser
+from dojo.models import Test
+
+
+class TestSpotbugsXMLParser(TestCase):
+
+    def test_no_findings(self):
+        parser = SpotbugsXMLParser(None, Test())
```

Comment:
Please don't keep this test with `None`. There doesn't make any sense.

Reply:
remove